modules/rosa-configure.adoc

```diff
@@ -0,0 +1,271 @@
+:system-module-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-get-started-cli.adoc
+
+[id="rosa-configure_{context}"]
+= Configuring `rosa`
+
+[role="_abstract"]
+Use the following commands to configure the `rosa` CLI.
+
+[id="rosa-login_{context}"]
+== login
+
+Logs in to your Red Hat account, saving the credentials to the `rosa` configuration file. You must provide a token when logging in. You can copy your token by from link:https://cloud.redhat.com/openshift/token/rosa[the {product-title} token page].
+
+`rosa` looks for a token in the following priority order:
+
+* Command-line arguments
+* The `ROSA_TOKEN` environment variable
+* The `rosa` configuration file
+* Interactively from a command-line prompt
+
+.Syntax
+[source,terminal]
+----
+$ rosa login [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--client-id
+|The OpenID client identifier(string). Default: `cloud-services`
+
+|--client-secret
+|The OpenID client secret (string).
+
+|--insecure
+|Enables insecure communication with the server. This disables verification of TLS certificates and host names.
+
+|--scope
+|The OpenID scope (string). If this option is used, it completely replaces the default scopes. This can be repeated multiple times to specify multiple scopes. Default: `openid`
+
+|--token
+|Accesses or refreshes the token (string).
+
+|--token-url
+|The OpenID token URL (string). Default: `\https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token`
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+|===
+
+[id="rosa-logout_{context}"]
+== logout
+
+Logs out of `rosa`. Logging out also removes the `rosa` configuration file.
+
+.Syntax
+[source,terminal]
+----
+$ rosa logout [arguments]
+----
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+|===
+
+[id="rosa-verify-permissions_{context}"]
+== verify permissions
+
+Verifies that the AWS permissions required to create a `rosa` cluster are configured correctly.
+
+.Syntax
+[source,terminal]
+----
+$ rosa verify permissions [arguments]
+----
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--region string
+|The AWS region in which to run. This value overrides the `AWS_REGION` environment variable.
```

Comment:
"in which to run" what? Does it mean to say, "The AWS region in which to run the command", or "The AWS region in which to run the cluster", or maybe something else?

Reply:
Fixed. It's "in which to run the command".